src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -175,22 +176,22 @@ public void syntheticStages() throws Exception {
 
         DepthFirstScanner scanner = new DepthFirstScanner();
 
-        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStageNames.toolInstall(), SyntheticContext.PRE)));
-        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStageNames.checkout(), SyntheticContext.PRE)));
-        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStageNames.postBuild(), SyntheticContext.POST)));
-        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStageNames.notifications(), SyntheticContext.POST)));
-        assertNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStageNames.dockerPull(), SyntheticContext.PRE)));
+        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStage.toolInstall(), SYNTHETIC_PRE)));
+        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStage.checkout(), SYNTHETIC_PRE)));
+        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStage.postBuild(), SYNTHETIC_POST)));
+        assertNotNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStage.notifications(), SYNTHETIC_POST)));
+        assertNull(scanner.findFirstMatch(heads, null, syntheticStagePredicate(SyntheticStage.dockerPull(), SYNTHETIC_PRE)));
     }
 
     private Predicate<FlowNode> syntheticStagePredicate(final String stageName,
-                                                        final SyntheticContext context) {
+                                                        final String context) {
         return new Predicate<FlowNode>() {
             @Override
             public boolean apply(FlowNode input) {
                 return input.getDisplayName().equals(stageName) &&
                         input.getAction(TagsAction.class) != null &&
```

Comment:
Wait.  Nevermind, that's test-only.  It would be a bug if it were a public API.  

Going to blame that one on still being jetlagged post-JenkinsWorld.